lib/core/constants.js

```diff
@@ -107,6 +107,25 @@ const headerNameLowerCasedRecord = {}
 // Note: object prototypes should not be able to be referenced. e.g. `Object#hasOwnProperty`.
 Object.setPrototypeOf(headerNameLowerCasedRecord, null)
 
+/**
+ * @type {Record<Lowercase<typeof wellknownHeaderNames[number]>, Buffer>}
+ */
+const wellknownHeaderNameBuffers = {}
```

Comment:
Use `Object.setPrototypeOf(headerNameLowerCasedRecord, null)` to set the null prototype so that the object prototype cannot be referenced.

Reply:
Please add tests like this.
https://github.com/nodejs/undici/blob/d6c44f3b49949dbdd66d9c56e729d503e6b09ddc/test/node-test/util.js#L118-L120

Reply:
Revisiting this and I don't think the actual record needs to be exported, however to the test it would need to be exported, wdyt?

Reply:
Actually, you don't have to add the test.